src/app/address/controllers/address.js

```diff
@@ -42,5 +35,23 @@ class AddressController extends BaseController {
       callback();
     });
   }
+
+  buildAddress(reqBody) {
+    const addressFlatNumber = reqBody.addressFlatNumber || null;
+    const addressHouseNumber = reqBody.addressHouseNumber || null;
+    const addressHouseName = reqBody.addressHouseName || null;
+    const addressStreetName = reqBody.addressStreetName;
+    const addressTownOrCity = reqBody.addressTownOrCity;
+    // const addressResidencyFrom = reqBody.addressResidencyFrom; // todo
```

Comment:
Just a note, as this will be a "Year from", it should be addressValidFrom and in the format "YYYY-01-01" to conform to the ISO standards we're using for the date

Reply:
Sounds good, this PR includes this as its just grabbing it from the UI. There will be logic here to put it into correct format. but this is good to know! TY